ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh

```diff
@@ -151,9 +152,22 @@ do
 
     extract_oc
 
-    mirror_image
+    if [[ "${ENABLE_OTA_TEST}" =~ "OCP-30832" ]]; then
+        MIRROR_RELEASE_IMAGE_REPO="${MIRROR_REGISTRY_HOST}/ota_auto/ocp"
+        mirror_apply_sig_flag="true"
```

Comment:
how about rename it to `mirror_apply_sig_together`?

Reply:
updated.